project/plugins.sbt

```
@@ -1,6 +1,6 @@
-resolvers += Resolver.sonatypeRepo("releases")
+resolvers += Resolver.sonatypeRepo("public")
 
-addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.24")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.27+36-56c9bf94-SNAPSHOT")
```

To be updated tomorrow
```suggestion
addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.28")
```